redbot/cogs/trivia/trivia.py

```diff
@@ -183,6 +185,73 @@ def __init__(self):
         else:
             await ctx.send(_("Done. I will no longer reward the winner with a payout."))
 
+    @triviaset.group(name="custom")
+    @commands.is_owner()
+    async def triviaset_custom(self, ctx: commands.Context):
+        """Manage Custom Trivia lists."""
+        pass
+
+    @triviaset_custom.command(name="list")
+    async def custom_trivia_list(self, ctx: commands.Context):
```

Comment:
0 lists should be special-cased:
![image](https://user-images.githubusercontent.com/6032823/77229758-df5b0100-6b8f-11ea-9493-3a735705d83b.png)